torch/utils/_cxx_pytree.py

```diff
@@ -32,7 +32,11 @@
 
 __all__ = [
     "PyTree",
-    "PyTreeSpec",
+    "Context",
+    "FlattenFunc",
+    "UnflattenFunc",
```

Comment:
Can we add a test that the `__all__` matches? Otherwise they will easily diverge